raster_loader/io/snowflake.py

```diff
@@ -0,0 +1,258 @@
+import json
+import rasterio
+import pandas as pd
+
+from typing import Iterable, List, Tuple
+
+from raster_loader.errors import (
+    IncompatibleRasterException,
+    import_error_snowflake,
+)
+
+from raster_loader.utils import ask_yes_no_question, batched
+
+from raster_loader.io.common import (
+    rasterio_metadata,
+    rasterio_windows_to_records,
+    get_number_of_blocks,
+    check_metadata_is_compatible,
+    update_metadata,
+)
+from raster_loader.io.datawarehouse import DataWarehouse
+
+try:
+    from snowflake.connector.pandas_tools import write_pandas
+    import snowflake.connector
+except ImportError:  # pragma: no cover
+    _has_snowflake = False
+else:
+    _has_snowflake = True
+
+
+class Snowflake(DataWarehouse):
```

Comment:
I would rename it as `SnowflakeConnection`

Reply:
Updated